R/nav-items.R

```diff
@@ -65,7 +65,7 @@ is_nav_item <- function(x) {
 #' @describeIn nav-items Adding spacing between nav items.
 #' @export
 nav_spacer <- function() {
-  div(class = "bslib-nav-spacer")
+  div(class = "bslib-nav-spacer", component_dependencies())
```

Comment:
This one seems a little out of place to be getting the complete bundle. Not that I object entirely but I'd also be in favor of pulling out the nav spacer dependency to stand on its own.